_posts/2020-03-11-sr2020-covid-19-updates.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: news
+title: COVID-19 Coronavirus Student Robotics Updates
+---
+
+As you will be aware, the COVID-19 situation is quickly evolving, having recently been delcared a pandemic. 
+Many large events are being cancelled and we want to be public about how we are responding to the latest news.
+
+## The Competition
+
+At the time of writing, we still intend to have the competition on the 18th and 19th April 2020, however are actively and carefully considering whether this needs to change.
+
+We will be contacting teams about our decision next week. We will also publish this on the Student Robotics website.
+
+Considering this is an event for minors, we will be taking decisions with an abundance of caution. 
+
+We are in contact with Reading University Students' Union who are continually reviewing the situation, and mitigating the risks associated with Coronavirus.
+
+### In the case of the competition going ahead
+
+We will be briefing all volunteers and attendees about best hygiene practices, both in advance and on the day, ensuring they're followed throughout the competition. 
+
+We expect everyone attending the competition to follow the [guidelines][phe-guidelines] on COVID-19 from Public Health England.
+
+### In the case of the competition being postponed
+
+We are considering weekends in late June / early July as alternatives for the competition. 
+We will be surveying team leaders to ensure we select the most appropriate date.
+
+## Tech Days
+
+At the time of writing, we still intend to run the London Tech Day on 28th March. However, we will contact all teams as soon as possible should this be cancelled.
+
+In the event of the competition being postponed, all planned Tech Days will be cancelled. 
+
+The [forum][forum] is still a great place to get help from blueshirts and other teams, in addition to showing off your progress!
```

Comment:
Maybe also mention remote mentoring?

Reply:
So we are continually accepting mentoring requests? Sounds good, will add

Reply:
Yeah, I don't think we'd turn anyone down if we had a volunteer who could help them.